### ISSUE TITLE
Show fiber counts in logarithmic scale?

### ISSUE THREAD

**Issue by soichih** [brainlife/app-wmaSeg#12]
I think we should show the fiber count in log scale?

![image](https://user-images.githubusercontent.com/923896/62888901-48cb8500-bd0e-11e9-8736-5eea973a8ae8.png)

To switch to log, all we need to do is to set the `type` field to "log" for plotlyjs

```
var layout = {
  xaxis: {
    type: 'log',
    autorange: true
  },
  yaxis: {
    type: 'log',
    autorange: true
  }
};

```

**Comment by DanNBullock** [comment]
I'm not sure that would be consistent with the goals of this visualization.  As I understand it this visualization is used to compare counts across subjects and tracts.  By log scaling it, you wash out all of the visually perceptible variation.  What would the goal of log-scaling be?

**Comment by soichih** [comment]
I don't know if there is a specific goal in mind for this visualization, but I'd argue that linear scaling is actually washing out the visually perceptible variation for tracts that has much small number of fibers (like IFOF). 

By the way, another way to show this might be to actually multiply the length of the fibers with the count of fibers (call it the "fiber size"). Would it be interesting to show that?

**Comment by francopestilli** [comment]
I like the idea of Log. Comparing across tracts currently is difficult because of the wide range in fiber number.

**Comment by DanNBullock** [comment]
Fair enough about using a log scale, but that' isn't necessarily something that needs to be done on this end.  Shouldn't you just be able to do that on the plot end?  In fact, I'd say we don't even need to make a choice on this, you could just have a button on the interface that would logscale the y axis.  Seems like it wouldn't be that hard to code.

Also, with respect to "By the way, another way to show this might be to actually multiply the length of the fibers with the count of fibers (call it the "fiber size"). Would it be interesting to show that?"

That's what I'd refer to as wiring cost or length.  Its one of the included outputs of the quality check.  I'd also argue that the quality check is something that we should be running standard on segmentation outputs.

**Comment by soichih** [comment]
plotlyjs currently doesn't provide the capability to switch between different scales.. but it might in the future.. 

We could still update it so that it will show in log scale *by default*. Right now we have to set it as part of generated plotlyjs json which is done in here > https://github.com/DanNBullock/wma_tools/blob/master/wma_formatForBrainLife_v2.m#L166

I *think* all you have to add is this

```
barlayout.xaxis.type = 'log';
```

**Comment by DanNBullock** [comment]
Would that leave the over displays as the absolute count?  If the hover display switches to log scale the visualization's utility will be greatly diminished.
